numpy/core/src/common/simd/avx2/arithmetic.h

```diff
@@ -72,4 +72,64 @@
 #define npyv_div_f32 _mm256_div_ps
 #define npyv_div_f64 _mm256_div_pd
 
+/***************************
+ * FUSED
+ ***************************/
+#ifdef NPY_HAVE_FMA3
+    // multiply and add, a*b + c
+    #define npyv_muladd_f32 _mm256_fmadd_ps
+    #define npyv_muladd_f64 _mm256_fmadd_pd
+    // multiply and subtract, a*b - c
+    #define npyv_mulsub_f32 _mm256_fmsub_ps
+    #define npyv_mulsub_f64 _mm256_fmsub_pd
+    // negate multiply and add, -(a*b) + c
+    #define npyv_nmuladd_f32 _mm256_fnmadd_ps
+    #define npyv_nmuladd_f64 _mm256_fnmadd_pd
+    // negate multiply and subtract, -(a*b) - c
+    #define npyv_nmulsub_f32 _mm256_fnmsub_ps
+    #define npyv_nmulsub_f64 _mm256_fnmsub_pd
+#elif defined(NPY_HAVE_FMA4)
+    // multiply and add, a*b + c
+    #define npyv_muladd_f32 _mm256_macc_ps
+    #define npyv_muladd_f64 _mm256_macc_pd
+    // multiply and subtract, a*b - c
+    #define npyv_mulsub_f32 _mm256_msub_ps
+    #define npyv_mulsub_f64 _mm256_msub_pd
+    // negate multiply and add, -(a*b) + c
+    #define npyv_nmuladd_f32 _mm256_nmacc_ps
+    #define npyv_nmuladd_f64 _mm256_nmacc_pd
```

Comment:
```suggestion
```
no need to translate FMA4 to FMA3 for AVX2

Reply:
since all AMD processors that support AVX2 also supports FMA3

Reply:
It can happen, you know if some tried to test the buggy FMA4 that kept hidden by AMD.
That can happen only via --cpu-baseline with gcc and clang since CPUID not gonna
report for FMA4.
.e.g. --cpu-baseline="avx2 fma4" instead of --cpu-baseline="avx2 fma3"

Reply:
I will just remove it, who cares about fma4 even AMD don't. it's enough to give support for it through sse.